packages/firestore/src/local/simple_db.ts

```diff
@@ -212,7 +217,21 @@ export class SimpleDb {
     return Number(version);
   }
 
-  constructor(private db: IDBDatabase) {}
+  constructor(private db: IDBDatabase) {
+    const iOSVersion = SimpleDb.getIOSVersion(getUA());
+    // NOTE: According to https://bugs.webkit.org/show_bug.cgi?id=197050, the
+    // bug we're checking for should exist in iOS >= 12.2 and < 13, but for
+    // whatever reason it's much harder to hit after 12.2 so we only proactively
+    // log on 12.2.
+    if (iOSVersion === 12.2) {
+      error(
+        'Firestore persistence on iOS 12.2 suffers from a bug in iOS 12.2 ' +
+          'Safari that may cause your app to stop working. See ' +
+          'https://stackoverflow.com/q/56496296/110915 for details ' +
+          'and potential workaround.'
```

Comment:
s/and potential/and a potential/ seems more grammatically correct.

Reply:
done